app/controllers/users_controller.rb

```diff
@@ -1,4 +1,12 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :correct_user, only: [:edit, :update]
+  before_action :admin_user, only: :destroy
+  
+  def index
+      @users = User.paginate(page: params[:page]) 
```

Comment:
```suggestion
    @users = User.paginate(page: params[:page]) 
```
as previously mentioned, try to stick to 2-space indentation, I'd strongly suggest setting this in you text editor so we don't always have to give the same feedback in future PRs